spiketoolkit/postprocessing/plot_features.py

```diff
@@ -0,0 +1,163 @@
+from plotly import graph_objs as go
```

Comment:
THis is super cool, but I think it should go in [`spikewidgets`](https://github.com/SpikeInterface/spikewidgets)

Reply:
(for now we are using matplotlib as backend in SW)

Reply:
I don't intend this to become part of `spiketoolkit` it's now there for my convenience, but I was planning on using this for my own analysis